<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Messager</class>
 <widget class="QMainWindow" name="Messager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>795</width>
    <height>601</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>795</width>
    <height>601</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Postogramm</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,2">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="1,11">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="find_sb_layout">
        <item>
         <widget class="QToolButton" name="open_sb_btn">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="search_t"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QListWidget" name="room_list"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QWidget" name="messaging_layout" native="true">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
      <layout class="QVBoxLayout" name="chat_layout" stretch="0,0,0">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <item>
        <layout class="QHBoxLayout" name="chat_info_layout">
         <item>
          <layout class="QVBoxLayout" name="n_a_u_layout">
           <item>
            <widget class="QLabel" name="chat_name_l">
             <property name="font">
              <font>
               <family>Noto Mono</family>
               <pointsize>12</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Chat &amp;Name</string>
             </property>
             <property name="textFormat">
              <enum>Qt::AutoText</enum>
             </property>
             <property name="scaledContents">
              <bool>false</bool>
             </property>
             <property name="margin">
              <number>0</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="chat_members_l">
             <property name="text">
              <string>Chat &amp;Members</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QToolButton" name="chat_search_btn">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="chat_info_btn">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="chat_menu_btn">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QListWidget" name="chat_list"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="messag_area">
         <item>
          <widget class="QLineEdit" name="message_box"/>
         </item>
         <item>
          <widget class="QToolButton" name="send_message">
           <property name="text">
            <string>&gt;</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>795</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuConnections">
    <property name="title">
     <string>&amp;Connections</string>
    </property>
    <addaction name="actionConnect"/>
    <addaction name="actionDisconnect_from_current"/>
    <addaction name="actionServer_list"/>
   </widget>
   <widget class="QMenu" name="menuSelf">
    <property name="title">
     <string>&amp;Self</string>
    </property>
    <addaction name="actionChange_password"/>
    <addaction name="actionChange_nickname"/>
    <addaction name="actionChange_username"/>
   </widget>
   <widget class="QMenu" name="menuPermissions">
    <property name="title">
     <string>&amp;Permissions</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>&amp;Tools</string>
    </property>
    <addaction name="actionChange_proxy"/>
    <addaction name="actionChange_username_2"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
   </widget>
   <addaction name="menuConnections"/>
   <addaction name="menuSelf"/>
   <addaction name="menuPermissions"/>
   <addaction name="menuTools"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionConnect">
   <property name="text">
    <string>&amp;Connect</string>
   </property>
  </action>
  <action name="actionDisconnect_from_current">
   <property name="text">
    <string>&amp;Disconnect from current</string>
   </property>
  </action>
  <action name="actionServer_list">
   <property name="text">
    <string>&amp;Server list</string>
   </property>
  </action>
  <action name="actionChange_password">
   <property name="text">
    <string>&amp;Change password</string>
   </property>
  </action>
  <action name="actionChange_nickname">
   <property name="text">
    <string>Change &amp;nickname</string>
   </property>
  </action>
  <action name="actionChange_username">
   <property name="text">
    <string>Change &amp;username</string>
   </property>
  </action>
  <action name="actionChange_proxy">
   <property name="text">
    <string>&amp;Change proxy</string>
   </property>
  </action>
  <action name="actionChange_username_2">
   <property name="text">
    <string>Change &amp;username</string>
   </property>
  </action>
  <action name="actionDisconnect">
   <property name="text">
    <string>Disconnect</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
